cclib/parser/gaussianparser.py

```diff
@@ -1201,6 +1207,21 @@ def extract(self, inputfile, line):
 
             self.moenergies = [numpy.array(x, "d") for x in self.moenergies]
 
+            # set homoenergies and lumoenergies
+            def find_mo_energy(mo,moenergies):
+                moenergy = []
+                for i in range(0,len(mo)):
+                    energies = moenergies[i]
+                    n = mo[i] - 1
+                    moenergy.append(energies[n])
+                    return moenergy
```

Comment:
ok so if I understand this correctly there might be multiple homo for multiple orbital energies. Why not make moenergies an array and then index it with mo values (-1)? Then you don't need a function right?

Reply:
so atm moenergies is a list of arrays, if you can make it a 2-d array then something like `moenergies_array[:, mo-1]` might do the trick?

Reply:
actually it would be something like this: `moenergies_array[np.arange(moenergies_array.shape[0]), mo-1]` assuming the moenergies is an array of x*y

Reply:
to be honest, I get the impression this is something they would rather leave out, based on the issue I posted. They've obviously considered that many users will want properties like this, and can easily extract them themselves, hence why they have the HOMO value at HOMO+1 so that users can do this without problems.

Reply:
stack list of arrays into a single multidimensional array, then one line for homoenergy and one for lumoenergy